share/spack/gitlab/cloud_pipelines/configs/ci.yaml

```diff
@@ -36,6 +36,34 @@ ci:
         - aws s3 sync --exclude "*" --include "*spec.json.sig*" /tmp ${SPACK_REMOTE_MIRROR_OVERRIDE}/build_cache
         - aws s3 cp /tmp/public_keys ${SPACK_REMOTE_MIRROR_OVERRIDE}/build_cache/_pgp --recursive --exclude "*" --include "*.pub"
 
+  - copy-job:
+      tags: ["service", "x86_64"]
+      image: "ghcr.io/spack/python-aws-bash:0.0.1"
+      before_script:
+      - - if [[ $CI_COMMIT_REF_NAME == "v"* ]]; then export SPACK_REPLACE_VERSION=$(echo "$CI_COMMIT_REF_NAME" | sed 's/\(v[[:digit:]]\+\.[[:digit:]]\+\).*/releases\/\1/'); fi
```

Comment:
This would match any branch that starts with `v`? Should probably use `CI_COMMIT_TAG`.

Reply:
That makes sense, will do.